rollup.config.js

```diff
@@ -35,48 +67,52 @@ const configs = ['posthog-node', 'posthog-web', 'posthog-ai'].reduce((acc, x) =>
           format: `es`,
         },
       ],
-      external,
-      plugins: [
-        // Allows node_modules resolution
-        resolve({ extensions }),
-        // Allow bundling cjs modules. Rollup doesn`t understand cjs
-        commonjs(),
-        json(),
-        // Compile TypeScript/JavaScript files
-        typescript({
-          include: [`*.(t|j)s+(|x)`, `**/*.(t|j)s+(|x)`],
-          tsconfig: `./${x}/tsconfig.json`,
-          sourceMap: true,
-        }),
-        babel({
-          extensions,
-          babelHelpers: 'bundled',
-          include: [`${x}/src/**/*`],
-          presets: [
-            ['@babel/preset-env', { targets: { node: 'current' } }],
-            '@babel/preset-typescript',
-            '@babel/preset-react',
-          ],
-        }),
-      ],
+      external: external(`./${x}/package.json`),
+      plugins: plugins(x),
     },
     {
-      input: `./${x}/lib/${x}/index.d.ts`,
+      input: `./${x}/index.ts`,
       output: [{ file: `./${x}/lib/index.d.ts`, format: 'es' }],
-      plugins: [
-        dts(),
-      ],
+      external: external(`./${x}/package.json`),
+      plugins: [resolve({ extensions }), dts({ tsconfig: `./${x}/tsconfig.json` })],
     },
   ]
 }, [])
 
-// Add submodule builds for posthog-ai
-const aiPkg = require('./posthog-ai/package.json')
-const aiExternal = [...globalExternal]
-  .concat(Object.keys(aiPkg.dependencies || {}))
-  .concat(Object.keys(aiPkg.peerDependencies || {}))
-  .concat(Object.keys(aiPkg.devDependencies || {}))
+const runtimes = ['node', 'edge']
+
+runtimes.forEach((runtime) => {
+  configs.push({
+    input: `./posthog-node/src/entrypoints/index.${runtime}.ts`,
+    output: [
+      {
+        file: `./posthog-node/lib/${runtime}/index.cjs.js`,
+        sourcemap: true,
+        exports: 'named',
+        format: 'cjs',
+      },
+      {
+        file: `./posthog-node/lib/${runtime}/index.esm.js`,
+        sourcemap: true,
+        format: 'es',
+      },
+    ],
+    external: external(`./posthog-node/package.json`),
+    plugins: plugins('posthog-node'),
+  })
+})
+
+// We only build types from node as all types should be the same
+configs.push({
+  input: `./posthog-node/src/entrypoints/index.node.ts`,
+  output: [{ file: `./posthog-node/lib/index.d.ts`, format: 'es' }],
+  external: external('./posthog-ai/package.json'),
```

Comment:
I'm guessing this is a typo @hpouillot?
```suggestion
  external: external('./posthog-node/package.json'),
```

Reply:
Yes, it should `posthog-node` here